pkg/env/env.go

```diff
@@ -7,16 +7,25 @@ import (
 	"bufio"
 	"fmt"
 	"os"
+	"path"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
 // DefaultEnvVariables set $PATH, $TERM and $container.
 var DefaultEnvVariables = map[string]string{
-	"PATH":      "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
-	"TERM":      "xterm",
-	"container": "podman",
+	"PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin",
+	"TERM": "xterm",
+}
+
+func init() {
+	// Set "container" to the current binary.
+	bin, err := os.Executable()
+	if err != nil {
+		panic(fmt.Sprintf("cannot determine executable: %v", err))
```

Comment:
Rather then panic could you make this a errors.Errorf and then set the id to "unknown"

Reply:
done